### ISSUE TITLE
Major security concern

### ISSUE THREAD

**Issue by charleshan** [aaronn/django-rest-framework-passwordless#33]
I know that the 6-digit login code length has been brought up before but the real problem is that the api checks the login code at a global level. This means that the larger the user base, the easier it becomes to get access to random accounts.

We should check the login code combined with the email.

Proposed change (pseudocode):
```
# serializers.py

callback_token = attrs.get('token', None)
token = CallbackToken.objects.get(key=callback_token, is_active=True)
```

to:
```
callback_token = attrs.get('token', None)
email = attrs.get('email', None)

user = User.objects.get(email=email)
token = CallbackToken.objects.get(key=callback_token, user=user, is_active=True)
```

**Comment by aaronn** [comment]
Hey, thanks for pointing this out. It seems reasonable– I am kind of jammed on work right now but I'm definitely open to a PR for this.

**Comment by aaronn** [comment]
Task here is abstracting a serializer and view that can take both `mobile` OR `email`– since we use the `/auth/token` endpoints. `/auth/verify/` can probably stay the same since it checks against the `request.user`.

**Comment by aaronn** [comment]
Fixed in 1.5.0
